lib/gems/pending/util/vmdb-logger.rb

```diff
@@ -108,9 +108,13 @@ def self.log_hashes(logger, h, options = {})
     filter = Array(options[:filter]).flatten.compact.map(&:to_s) << "password"
     filter.uniq!
 
-    YAML.dump(h).split("\n").each do |l|
+    values = YAML.dump(h).gsub(MiqPassword::REGEXP) { |m| m.gsub($2, "FILTERED") }
```

Comment:
The substitution here ends up looking like `v2:{FILTERED}` where the braces come from the encrypted string.  Adding `[...]` into the mix seemed useless.